_posts/2017-01-28-mfem-3.3.md

```diff
@@ -5,7 +5,7 @@ tags: new-release
 
 Version 3.3 of MFEM, a lightweight, general, scalable C++ library for finite element methods, is now available at: [http://mfem.org](http://mfem.org)
 
-The goal of MFEM is to enable high-performance scalable finite element discretization research and application development on a wide variety of platforms, ranging from laptops to [exascale supercomputers](https://exascaleproject.org/2016/11/11/ecp_co-design_centers).
+The goal of MFEM is to enable high-performance scalable finite element discretization research and application development on a wide variety of platforms, ranging from laptops to exascale supercomputers.
```

Comment:
Would it make sense to still link to the exascale project as a whole?
https://www.exascaleproject.org/